lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -68,6 +68,15 @@ def create_document_symbol_listener(response_builder, dispatcher)
         DocumentSymbol.new(response_builder, dispatcher)
       end
 
+      sig { params(changes: T::Array[{ uri: String, type: Integer }]).void }
+      def workspace_did_change_watched_files(changes)
+        if changes.any? do |change|
+             change[:uri].end_with?("db/schema.rb") || change[:uri].end_with?("structure.sql")
```

Comment:
Normally we would only have to support `structure.sql` but in `Shopify/shopify` we have additional structure files.